cmd/notation/cert.go

```diff
@@ -316,17 +320,17 @@ func listCerts(opts *certListOpts) error {
 }
 
 func showCerts(opts *certShowOpts) error {
-	storeType := opts.storeType
+	storeType := strings.TrimSpace(opts.storeType)
```

Comment:
I might be wrong, looks like empty check on `storeType` and `namedStore` is required in all commands. Maybe we can move it to a method of `certShowOpts`.